docs/DeveloperGuide.md

```diff
@@ -8,22 +8,72 @@
 
 {Describe the design and implementation of the product. Use UML diagrams and short code snippets where applicable.}
 
+### Update patient information
+Step 1:Patient inform the nurse of their change in personal information. Nurse will open up the Duke system and request to submit the change. Duke will call the Parser#etPatientInfo() to update the patient information. 
+
+Step 2:
+The parser will call the database storing patient information to update the change.
+
+The flow is presented in the sequence diagram (will be included later after pull request is permitted):
+
+
+### Exit program
+Step 1: User type "bye" as input to exit the program. IHospital will call Parser#getCommand()
+to return the user command received.
+
+Step 2: After checking the user command is "bye", IHospital will
+call Ui#sayGoodbye to print Goodbye message
+
+The following sequence diagram shows how the exit operation works:
+
+![](ByeCommand.png)
+
+### Add Doctor
+Step 1: User type add command to add a doctor into the system and provide information required.
+IHospital will call Parser#getCommand() to return the user command received, it will then call
+Parser#parse to understand that User wants to add a doctor, and Parser#parse will return an 
+AddDoctorCommand object.
+
+Step 2: IHospital will call execute of the AddDoctorCommand object and AddDoctorCommand will
+create a new Doctor with the given information and add it into DoctorList using 
+DoctorList#addDoctor(doctor).
+
+The following sequence diagram shows how the add operation works:
+
+![](AddDoctor.png)
```

Comment:
![image](https://user-images.githubusercontent.com/15943697/160964494-e9c00116-127f-4661-92d3-20cf56ce4b1c.png)
Similarly for here, the activation bars are missing.
![image](https://user-images.githubusercontent.com/15943697/160964533-394c4815-c5af-44b2-a131-addfab9d12a3.png)
Similar mistakes as the above diagram, where the return arrow should have the datatype.

![image](https://user-images.githubusercontent.com/15943697/160965138-1df6f048-fefa-41a9-b350-bf3fb7d2c327.png)
Similarly, should remove the bottom boxes as well.

Perhaps you could have a line break for long comments so that you diagram would not appear very long. As the current diagram is a little difficult to read.